scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/RestPageParameters.java

```diff
@@ -0,0 +1,114 @@
+package org.ow2.proactive.scheduler.core;
+
+import org.ow2.proactive.scheduler.common.exception.NotConnectedException;
+import org.ow2.proactive.scheduler.common.exception.PermissionException;
+
+
+class RestPageParameters {
+
+    private SchedulerFrontendState frontEnd;
+    private String calledMethod;
+    private long from;
+    private long to;
+    private boolean myTasksOnly;
+    private boolean running;
+    private boolean pending;
+    private boolean finished;
+    private int offset;
+    private int limit;
+    private String tag;
+
+    public RestPageParameters(SchedulerFrontendState frontEnd, String calledMethod, long from, long to,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 11 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)